app/services/calendar_service.rb

```diff
@@ -0,0 +1,40 @@
+class CalendarService
+
+  def self.post_events(user, title, description, date)
+    response = conn.post('event/new?') do |req|
+      req.params['token'] = user.google_token
+      req.params['refresh_token'] = user.google_refresh_token
+      req.params['calendar_id'] = user.calendar_id
+      req.params['name'] = title
+      req.params['description'] = description
+      req.params['date'] = date
+    end
+    return 0 unless JSON.parse(response.body, symbolize_names: true).empty?
```

Comment:
This will be removed in the next PR. I meant to use the `0` status to add a conditional in the `plant_garden` controller.

Reply:
sounds good. good catch